### PR TITLE
fix(test): fixing the failing tests

### PR DESCRIPTION
**Pull request by pranshuchittora**
- Fixed
    - test/config-lookup/dotfolder-single/.webpack/webpack.config.js
    - test/output/defaults/output-defaults.test.js
    - test/source-map/array/source-map-array.test.js

**What kind of change does this PR introduce?**

Fix - Tests

**Did you add tests for your changes?**
Yes

**If relevant, did you update the documentation?**
No 

**Summary**
Fixing the failing tests

**Does this PR introduce a breaking change?**

**Other information**

### CODE REVIEW

File: test/output/defaults/output-defaults.test.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ describe('output flag defaults', () => {
         });
     });
     it('set default output directory on empty flag', done => {
-        const { stdout, stderr } = run(__dirname, ['--entry', './a.js', '--output'], false);
+        const { stdout, stderr } = run(__dirname, ['--entry', './a.js', '--output', './dist/bundle.js'], false);
```
Comment (rishabh3112) [inline test/output/defaults/output-defaults.test.js]: It violates the test statement : "set default output directory on empty flag"

Comment (pranshuchittora) [reply test/output/defaults/output-defaults.test.js]: Then the output will be `main.js` not `bundle.js`.I think changing it to `main.js` would be a feasible approach ??

File: package.json [hunk 2]
```diff
@@ -38,10 +38,10 @@
...
     "reportCoverage": "nyc report --reporter=json && codecov -f coverage/coverage-final.json --disable=gcov",
     "smoketest": "./smoketests/smoketests.sh",
-    "test": "nyc jest --maxWorkers=4 --reporters=default --reporters=jest-junit",
-    "test:cli": "nyc jest test/ --maxWorkers=4 --reporters=default --reporters=jest-junit",
-    "test:packages": "nyc jest packages/ --maxWorkers=4 --reporters=default --reporters=jest-junit",
-    "test:ci": "nyc jest --maxWorkers=$(nproc) --reporters=default --reporters=jest-junit --colors",
+    "test": "nyc jest --maxWorkers=4 --reporters=default --reporters=jest-junit --no-colors",
+    "test:cli": "nyc jest test/ --maxWorkers=4 --reporters=default --reporters=jest-junit  --no-colors",
+    "test:packages": "nyc jest packages/ --maxWorkers=4 --reporters=default --reporters=jest-junit  --no-colors",
+    "test:ci": "nyc jest --maxWorkers=$(nproc) --reporters=default --reporters=jest-junit --no-colors",
```
Comment (rishabh3112) [inline package.json]: Colors are needed because `webpack-cli` has a `--color` flag that enables and disables colors in the help.

**Comment by pranshuchittora** [comment]
> `output-defaults` needs to have the `--defaults` flag, I think

Oops, still failing. Few roadblocking issues are 
- Merge not working
- CLI flags not overwriting the config

File: package.json (hunk 2 above)
Comment (pranshuchittora) [reply package.json]: Yeah, the CLI output is handled by reading the stdout stream, hence disabling the colour will disable colours only for the jest view part. Hence disabling it won't affect any of the tests related to colour characters.

Screenshot from 2019-12-07 23-13-01

Comment (pranshuchittora) [reply package.json]: :arrow_up_small: in spite of disabling the colour, the logs can have colours.

Comment (rishabh3112) [reply package.json]: @pranshuchittora Thanks for insight 👍

**Comment by anshumanv** [comment]
Tests are passing now, this can be closed.

**Comment by ematipico** [comment]
Thank you @anshumanv !

**Pull request closed without merging**